src/game/savegame/savegame.c

```diff
@@ -509,7 +515,9 @@ void Savegame_ScanSavedGames(void)
 
                 char *full_path =
                     Memory_Alloc(strlen(SAVES_DIR) + strlen(filename) + 2);
-                sprintf(full_path, "%s/%s", SAVES_DIR, filename);
+                size_t out_size =
+                    snprintf(NULL, 0, "%s/%s", SAVES_DIR, filename) + 1;
+                snprintf(full_path, out_size, "%s/%s", SAVES_DIR, filename);
```

Comment:
Pointless `snprintf` – we just calculated how much memory to allocate above, so the buffer is guaranteed to have enough capacity.

Reply:
Ok. I was just gonna standardize to `snprintf` to ensure we don't run over the buffer size and ensure null termination. But I will change it back.